### PR TITLE
Game statistics

### PR DESCRIPTION
**Pull request by elyhess**
Add game statistics methods with passing tests.

### CODE REVIEW

File: lib/game_manager.rb [hunk 1]
```diff
@@ -22,4 +22,79 @@ def all
...
+
+  def average_goals_per_game
+    sum = @games.sum do |game|
+      game.away_goals + game.home_goals
+    end
+
+    (sum.to_f / @games.size).round(2)
+  end
+
+  def goal_count_by_season(season)
```
Comment (philmccarthy) [inline lib/game_manager.rb:83]: let's rename this helper method to `goal_count(season)` to save a few characters

File: lib/game_manager.rb [hunk 2]
```diff
@@ -22,4 +22,79 @@ def all
...
+    games_by_season = @games.select do |game|
+      season == game.season
+    end
+    games_by_season.sum do |game|
+      game.away_goals + game.home_goals
+    end
+  end
+
+  def average_goals_by_season
+    average_by_season = {}
```
Comment (philmccarthy) [inline lib/game_manager.rb:93]: shorten `average` to `avg` here and on lines 95 & 97 so line 95 isn't so long

File: lib/game_manager.rb [hunk 3]
```diff
@@ -22,4 +22,79 @@ def all
...
+    @games.each do |game|
+      average_by_season[game.season] = (goal_count_by_season(game.season) / game_count(game.season).to_f).round(2)
```
Comment (philmccarthy) [inline lib/game_manager.rb:95]: match renamed `goal_count(season)` method from above

**Review by philmccarthy — CHANGES_REQUESTED** [review]

File: lib/game_manager.rb (hunk 3 above)
Comment (elyhess) [reply lib/game_manager.rb]: Added all above changes and adjusted tests so that all tests pass w/ new variable names

**Review by philmccarthy — APPROVED** [review]

**Pull request merged**